auth/templates.go

```diff
@@ -68,14 +70,12 @@ func renderTemplate(w http.ResponseWriter, tmpl *template.Template, data interfa
 		return
 	}
 
-	switch err := err.(type) {
-	case *template.Error:
+	var execErr txttpl.ExecError
```

Comment:
i am to stupid how you figured out that this error is of type ExecError. But i think it is correct

Reply:
My comment above this line stated that only ExecErrors are safe in terms of not having partial writes, so I dug in the template code and found this comment (again):

/usr/local/go/src/text/template/exec.go:114
// ExecError is the custom error type returned when Execute has an
// error evaluating its template. (If a write error occurs, the actual
// error is returned; it will not be of type ExecError.)